tests/unit/index.test.js

```diff
@@ -1,5 +1,27 @@
-import { describe, expect, it } from "vitest";
+import { describe, expect, it, test } from "vitest";
+import { shortAddr } from '/src/shared/lib/utils';  // Error from Vitest: ReferenceError: window is not defined
 
+
+// Unit-tests to verify "shortAddr(addr)" function accepts only a string as "addr" argument
+describe("shortAddr", () => {
+	it("Positive test cases - if 'addr' argument is a string", () => {
+		expect(shortAddr("string")).not.toThrow();
```

Comment:
The ﻿`expect()` function, when used with the ﻿`toThrow()` matcher, requires a function as its argument. The current code, however, passes a string, which is the result of invoking a function, instead of passing the function itself. 
To correct this, you should enclose your function invocation within an arrow function or use the ﻿bind method to attach the parameter to the function. For example:
```
expect(() => shortAddr("234")).not.toThrow();
expect(shortAddr.bind(this, "234")).not.toThrow(); // This is a somewhat old-school approach
```

Reply:
Thanks! 
Done.